readme.md

```diff
@@ -0,0 +1,57 @@
+## Connecting a repository
+
+In order to connect a repository with blog articles, you have to link it via `app:add-blog`. You also have to add a
+commit hook in your repository with the secret provided. This makes sure that both the initial creation and additional
+updates in the repository will sync it to the app.
+
+
+### Repository Layout
+
+The main file you need is the `blog.yml`, this contains the settings of your blog. An example below:
+
+```yml
+author:
+    name:  Henk de Vries
+    email: henk@devries.com
+    urls:
+        github:   https://.../
+        facebook: https://.../
+        twitter:  https://.../
+
+settings:
+    introduction: author.md
+    drafts: /drafts/
+    published: /published/
+```
+
+Additional settings you can apply:
+ - `introduction`: links to your intro which will be displayed on the right side next to posts.
+ - `drafts`: which indicates your draft blog posts directory.
+ - `published`: which indicates your published blog posts directory.
+
+
+### Blog Post Statuses
+
+There are three different statuses a blog post can have:
+ - Draft: will be displayed when the url is accessed directly but not listed.
+ - Unpublished: same as draft but in the published directory and will be released if the publish data is passed.
```

Comment:
Maybe:
same as draft but in the published directory but have a date in the future. Once the date has been passed they will become published